string arm_name
sensor_msgs/PointCloud2 cluster
string collision_name
